### PR TITLE
Set the broadcast address by calculating it with the IP and SN of Ethernet lib

### PR DESCRIPTION
**Pull request by Captnlink**
I Followed this calculator : http://jodies.de/ipcalc to calculate the broadcast address and applied it to the ArtNet.begin(...) so it can be set automatically.

### CODE REVIEW

**Comment by Captnlink** [comment]
Maybe put the version in library.json to 1.0.1?

**Comment by natcl** [comment]
This looks nice but I wouldn't include the setBroadcast in artnet.begin as this will break the library on microcontrollers that use wifi (ESP32, 8266).
Perhaps we should leave it as an optional part of the sketch to allow more flexibility when the Ethernet lib is not used ?

**Comment by Captnlink** [comment]
Would it be better to put the IP and Subnet as parameters to remove the dependency to Ethernet and give some leeway for WiFi uses?

**Comment by natcl** [comment]
I have the feeling artnet.begin shouldn't wrap ethernet.begin as it causes confusion most of the time. we can discuss it tomorrow !

**Comment by Captnlink** [comment]
With this I think it should be usable with either Wifi and Ethernet.

File: Artnet.h [hunk 1]
```diff
@@ -99,6 +99,7 @@ class Artnet
 
   void begin(byte mac[], byte ip[]);
   void begin();
+  void setBroadcast();
```
Comment (natcl) [inline Artnet.h]: Shouldn't this be setBroadcastAuto ?

Comment (natcl) [reply Artnet.h]: with IPAddress ip, IPAddress sn ?

Comment (Captnlink) [reply Artnet.h]: Yes you are totally right. I will change it to : void setBroadcastAuto(IPAddress ip, IPAddress sn);

**Pull request merged**